pkg/scheduler/queue/queue.go

```diff
@@ -108,9 +108,9 @@ func NewRequestQueue(maxOutstandingPerTenant int, forgetDelay time.Duration, que
 		stopCompleted: make(chan struct{}),
 
 		// These channels must not be buffered so that we can detect when dispatcherLoop() has finished.
-		querierOperations:           make(chan querierOperation),
-		enqueueRequests:             make(chan enqueueRequest),
-		availableQuerierConnections: make(chan *querierConnection),
+		querierOperations:             make(chan querierOperation),
+		enqueueRequests:               make(chan enqueueRequest),
```

Comment:
`enqueueRequest`/`enqueueRequests` are similarly verbs

the channel could be `requestsToEnqueue` or `requestsToDispatch`

the type could just be a private `request` type or `schedulerRequest` or `tenantRequest` or `queryRequest`, or `schedulerDispatchRequest`, or even `schedulerEnqueueRequests`, which still has the verb but at least clarifies which queue we are talking about

Reply:
I like `requestsToEnqueue`, thanks for the suggestion. I'll change it now.